### PR TITLE
update k gs template cluster doc

### PR DESCRIPTION
**Pull request by vxav**
### What does this PR do?

(Please set a descriptive PR title. Use this space for additional explanations.)

### What does it look like?

(If this is more than a textual change, a screenshot can help in some cases to speed up the review process.)

### Any background context you can provide?

(Please link public issues or summarize if not public.)

### Have you maintained the front matter?

(Please bump the last_review_date in case this qualifies as a review for an entire page. Provide user_questions which should be answered in the page. Provide a meaningful description.)

### CODE REVIEW

File: src/content/use-the-api/kubectl-gs/template-cluster.md [hunk 1]
```diff
@@ -658,6 +704,112 @@ status:
...
+    metadata:
+      description: Development Cluster
+      name: dev01
+      organization: acme
+    providerSpecific:
+      location: germanywestcentral
+      subscriptionId: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
```
Comment (pipo02mix) [inline src/content/use-the-api/kubectl-gs/template-cluster.md:729]:
```suggestion
```

File: src/content/use-the-api/kubectl-gs/template-cluster.md [hunk 2]
```diff
@@ -658,6 +704,112 @@ status:
...
+  version: 0.0.29
+---
+apiVersion: v1
+data:
+  values: |
+    clusterName: dev01
+    organization: acme
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
```
Comment (pipo02mix) [inline src/content/use-the-api/kubectl-gs/template-cluster.md:773]:
```suggestion
```

File: src/content/use-the-api/kubectl-gs/template-cluster.md [hunk 3]
```diff
@@ -658,6 +704,112 @@ status:
...
+    secret:
+      name: ""
+      namespace: ""
+  kubeConfig:
+    context:
+      name: ""
+    inCluster: true
+    secret:
+      name: ""
+      namespace: ""
```
Comment (pipo02mix) [inline src/content/use-the-api/kubectl-gs/template-cluster.md:803]:
```suggestion
  kubeConfig:
    inCluster: true
```
simplifying

File: src/content/use-the-api/kubectl-gs/template-cluster.md (hunk 2 above)
Comment (vxav) [reply src/content/use-the-api/kubectl-gs/template-cluster.md:773]: This is what the command generates actually

File: src/content/use-the-api/kubectl-gs/template-cluster.md (hunk 1 above)
Comment (vxav) [reply src/content/use-the-api/kubectl-gs/template-cluster.md:729]: This is what the command generates actually

File: src/content/use-the-api/kubectl-gs/template-cluster.md (hunk 3 above)
Comment (vxav) [reply src/content/use-the-api/kubectl-gs/template-cluster.md:803]: We still need `kubeConfig.inCluster=true`

**Review by AndiDog — APPROVED** [review]

**Pull request merged**